<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualFileManager</class>
 <widget class="QMainWindow" name="VisualFileManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VisualFileManager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_Cd">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Cd</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Up">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Paste">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Paste</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Copy">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Exit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>460</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Menu">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Menu&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lwt_File">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>421</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Create">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Root">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Root</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="let_Menu">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Delete">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
